contracts/axelar-gas-service/src/test.rs

```diff
@@ -81,9 +159,57 @@ fn pay_gas_for_contract_call() {
     );
 }
 
+#[test]
+fn fail_collect_fees_invalid_amount() {
+    let (env, contract_id, gas_collector, client) = setup_env();
+    client.initialize(&gas_collector);
+
+    let token_address: Address = env.register_stellar_asset_contract(Address::generate(&env));
+
+    let supply: i128 = 1000;
+    // refund_amount <= 0
```

Comment:
Maybe update comment (adding more explanation) or remove it?

Reply:
added an explanation of the test as a docstring and removed the comment